package com.pavani.focusquest.repository;
import com.pavani.focusquest.entity.Task;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.List;
public interface TaskRepository extends JpaRepository<Task,Long>{ List<Task> findByUserIdOrderByCreatedAtDesc(Long userId); }
